pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -1012,7 +1037,7 @@ func TestReconcile(t *testing.T) {
 			Object: activeKPAMinScale(overscale, defaultScale),
 		}},
 	}, {
-		Name: "scaled-to-0-no-scale-data",
+		Name: "scaled to 0 no scale data",
```

Comment:
dahes are cool :-)

Reply:
😆 I saw the other tests didn't have dashes so I removed the dashes in this one